src/screens/Ticketing/Purchase/utils.ts

```diff
@@ -14,21 +21,42 @@ export type PurchaseFlow = {
    * ticket.
    */
   travelDateSelectionEnabled: boolean;
+
+  /**
+   * Whether the customer should be able to select between a set of durations
+   * for the ticket, based on `fareproduct.durationDays`.
+   */
+  durationSelectionEnabled: boolean;
 };
 
 export const getPurchaseFlow = (
   product: PreassignedFareProduct,
```

Comment:
maybe just pass the type here as a prop?

Reply:
Good call! Updated now.